installation/docker-containers/high-availability-install/README.md

```diff
@@ -0,0 +1,68 @@
+# Guide to install Rocket.Chat as HA with mongodb replicaset as backend
+
+## Install MongoDB Replicaset
+
+- Use <https://medium.com/@gargar454/deploy-a-mongodb-cluster-in-steps-9-using-docker-49205e231319#.8dfln7n8c> to setup the mongodb replicaset
+- Create oplogger user with rights on local oplog database:
+    `db.createUser({user: "oplogger", pwd: "password", roles: [{role: "read", db: "local"}]})`
+- Create rocket user to authenticate with rocketchat database:
+    `db.createUser({user: "rocket", pwd: "password", roles: [{role: "readWrite", db: "rocketchat"}]})`
+
+## Create rocket.chat docker container
+
+- Use example docker-compose.yaml in repo
+- Fill the INSTANCE_IP env with the local ip of the docker-host (in a 3 node setup this is unique on every host)
+- Fill the passwords from rocket and oplogger
+- Change the IP of every host in the example docker-compose.yaml -> extra_hosts inserts host entrys so each rocketchat-container could resolve the others by the name
+- Be sure that the instances could communicate on port 3000 with each other, this is important to see messages on other hosts directly
+
+```
+rocketchat:
+    image: rocketchat/rocket.chat:latest
+    environment:
+        - PORT=3000
+        - ROOT_URL=https://chat.domain.de
+        - MONGO_URL=mongodb://rocket:password@rocket-1:27017,rocket-2:27017,rocket-3:27017/rocketchat?replicaSet=rs0&readPreference=nearest&w=majority
+        - MONGO_OPLOG_URL=mongodb://username:password@rocket-1:27017,rocket-2:27017,rocket-3:27017/local?authSource=admin&replicaSet=rs0
+        - INSTANCE_IP=<ip of the local instance>
+    ports:
+        - 3000:3000
+    extra_hosts:
+        - "rocket-1:10.250.250.13"
+        - "rocket-2:10.250.250.14"
+        - "rocket-3:10.250.250.17"
+```
+
+### Create Loadbalancer
+
+- Now setup up a reverse proxy on each host to terminate ssl on each rocket.chat node (use example apache2 config in repo) or terminate ssl on the loadbalancer, this is up to you
```

Comment:
apache2 here a preference? Apache2 has tendencies to have issues with websockets.  Seems to not always come enabled out of box.  So can be troublesome.

Also if using proxy on each machine.. might want to make sure to note they will need to add DNS A record for each one.  and load balancing will actually be done via DNS.

Typically in most tutorials we put something like nginx in front of multiple and load balance there.

Reply:
@MartinSchoeler I think I would simplify this and link to our reverse proxy docs and let them pick their own, that way we don't have to update apache config in multiple places.

Reply:
I agree with Aaron, can you point to our reverse proxy docs.